nym-vpn-x/src-tauri/src/main.rs

```diff
@@ -36,13 +36,15 @@ mod gateway;
 mod grpc;
 mod network;
 mod states;
+mod system_tray;
 mod vpn_status;
 mod window;
 
 pub const APP_DIR: &str = "nym-vpn-x";
 const APP_CONFIG_FILE: &str = "config.toml";
 const ENV_APP_NOSPLASH: &str = "APP_NOSPLASH";
 const VPND_RETRY_INTERVAL: Duration = Duration::from_secs(2);
+const SYSTRAY_ID: &str = "main";
```

Comment:
What is id used for? Will it clash with other apps?

Reply:
no → https://docs.rs/tauri/latest/tauri/struct.SystemTray.html#method.with_id

Reply:
That link doesn't really answer the question though ...